### ISSUE TITLE
Python (and other code?) syntax highlighting in Software Carpentry markdown

### ISSUE THREAD

**Issue by emiliom** [geohackweek/vector#3]
@lsetiawan Let's use github issues to follow up on specific, well, issues (instead of sprawling emails).

Last year I couldn't get the Software Carpentry markdown templates to generate syntax highlighted Python code. eg, see https://geohackweek.github.io/vector/04-geopandas-intro/

Is that a problem you heard from others, that has been addressed or identified? If not, I think I mentioned it to @ocefpaf and he said something constructive back then. I'd love to have this fixed this time around.

I'm referring to the analog of using this in Markdown to get syntax highlighting for Python (or any other language) -- here I'm artificially using "#" to start every line, so that all characters show up:
```
# ```python
# # My python code
# a = b + round(c)
# ```
```

That markdown did not get converted into syntax-highlighted code in the Software Carpentry pages. Last year I also looked into other mechanisms for getting syntax highlighting in those templates, including this one:
```
{% highlight python %}
import xarray as xr
{% endhighlight %}
```
Nothing worked.

*(There's no rush to address this problem right away! I just wanted to lay it out now while we still have plenty of time)*

**Comment by emiliom** [comment]
email follow-ups from Don:

> I asked Anthony if he knows anything about the syntax highlighting in software carpentry, and he also haven’t heard of any updates on that front. I will see if I can find any solution.

Thanks.

> This is what I got from the software carpentry website https://swcarpentry.github.io/lesson-example/aio/:
> **Why No Syntax Highlighting?** We do not use syntax highlighting for code blocks because some learners’ systems won’t do it, or will do it differently than what they see on screen.

Do you know what they mean by "learners' systems"? Is that just web browsers?? Or stuff like rendering for the visually impaired?

**Comment by lsetiawan** [comment]
Resolved by #11 and https://github.com/geohackweek/vector/commit/56937c9eca6314c85c1314da9d36f5b309089203. Closing this issue.
